src/cascadia/TerminalApp/HasNestedCommandsVisibilityConverter.cpp

```diff
@@ -36,4 +39,23 @@ namespace winrt::TerminalApp::implementation
     {
         throw hresult_not_implemented();
     }
+
+    Foundation::IInspectable HasNestedCommandsHelpTextConverter::Convert(Foundation::IInspectable const& value,

```

Comment:
are we seriously adding another converter
for the life of me
come on

Reply:
I hate it so much. But this was the workaround I was able to think of to get access to the `ListViewItem`. If we can somehow get access to that item, a simple `AutomationProperty::SetHelpText()` call would do all of this for us.